examples/rest/custom-json-get.py

```diff
@@ -0,0 +1,7 @@
+from polygon import RESTClient
+from examples import custom_json
```

Comment:
You may have to add a `# type: ignore` before this line to fix the `mypy` linting.